### PR TITLE
Added package zaier.

### PR DESCRIPTION
**Pull request by ZaierAouani**
Please review package "zaier". Thanks!

### CODE REVIEW

**Review by tsjuder1 — APPROVED** [review]
Looks splendid

File: src/test/java/Group4/zaier/JavaPractice.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package Group4.zaier;
+
+public class JavaPractice {
+    public static void main(String[] args) {
+        //started TC1
+        //java practice
+        String[] arr = {"splunk","makes","perfect","product","makes"};
+        String w1 = "product";
+        String w2 = "z";
```
Comment (BadamkhandB) [inline src/test/java/Group4/zaier/JavaPractice.java:9]: nice work

**Review by BadamkhandB — APPROVED** [review]

**Review by KarinaFrei — APPROVED** [review]
Good job.

**Review by VeronikaKrasnokutska — APPROVED** [review]
Keep going.Great job

**Review by natebell510 — APPROVED** [review]
ok

**Pull request merged**